pkg/app/piped/platformprovider/ecs/client.go

```diff
@@ -470,6 +494,68 @@ func (c *client) ModifyListeners(ctx context.Context, listenerArns []string, rou
 	return nil
 }
 
+func (c *client) ModifyListenerOrRule(ctx context.Context, listenerRuleArns []string, routingTrafficCfg RoutingTrafficConfig) error {
```

Comment:
This is not being used, should we remove this?

Reply:
Thank you,Correspondence
[3a4bea5](https://github.com/pipe-cd/pipecd/pull/4668/commits/3a4bea5ab8f0f8baf79ea9734b6a91563c9424e8)